lib/issues.go

```diff
@@ -47,10 +43,11 @@ func CompareIssues(config cfg.Config, ghClient clients.GitHubClient, jiraClient
 	for _, ghIssue := range ghIssues {
 		found := false
 		for _, jIssue := range jiraIssues {
-			id, _ := jIssue.Fields.Unknowns.Int(config.GetFieldKey(cfg.GitHubID))
-			if int64(*ghIssue.ID) == id {
+
+			id, _ := config.GetFieldMapper().GetFieldValue(&jIssue, cfg.GitHubID)
```

Comment:
This looks like it is ignoring any returned error. Is that OK?

Reply:
Nope. This was just to get from point A to point B quickly. There are actually many places throughout the codebase that are not handling errors (my IDE underlines them in yellow). I was planning to go back through and update that across the codebase later.

Reply:
Sounds good.